move_base/include/move_base/move_base.h

```diff
@@ -226,7 +226,9 @@ namespace move_base {
 
       geometry_msgs::PoseStamped goalToGlobalFrame(const geometry_msgs::PoseStamped& goal_pose_msg);
 
-      void publishFeedback(const geometry_msgs::PoseStamped& current_position);
+      void publishFeedback(const geometry_msgs::PoseStamped& current_position, const geometry_msgs::PoseStamped& goal);
```

Comment:
Can we name this smth like `computeAndPublishFeedback`? 
Having methods named identically but doing different things makes it terrible to understand.
Otherwise, this PR is ok.

Reply:
Sure, thanks